filebeat/docs/reference/configuration.asciidoc

```diff
@@ -19,7 +19,7 @@ configuration settings, you need to restart {beatname_uc} to pick up the changes
 * <<redis-output>>
 * <<file-output>>
 * <<console-output>>
-* <<configuration-output-tls>>
+* <<configuration-output-ssl>>
```

Comment:
Just a note that when we change IDs for topics, we need to ask the web team to set up redirects, or our links from forum topics will be broken. That's a good reason to avoid renaming IDs unless (which is probably true in this case) you definitely want the name to change, and it's worth the extra work to get the redirect set up.